whatshap/cli/stats.py

```diff
@@ -194,6 +198,46 @@ def add_heterozygous_variants(self, variants: int):
     def add_heterozygous_snvs(self, snvs: int):
         self.heterozygous_snvs += snvs
 
+    def get_nonoverlapping_blocks(self) -> List[PhasedBlock]:
+        """Split phase blocks into nonoverlapping subblocks"""
+        split_blocks = []
```

Comment:
Please move this line to just above the `while`: This function does two things 1) preprocess blocks, 2) split blocks. Logically, this line belongs to the second part.

Reply:
Yes, that makes more sense